uwrt_mars_rover_hw/drivers/roboteq_driver/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
delete this file. since integration with ros, you should be building this with `catkin_tools` now